### ISSUE TITLE
:results value not working; :results output working

### ISSUE THREAD

**Issue by rebcabin** [gjkerns/ob-julia#17]
I have julia installed by a symbolic link in /usr/local/bin/. I have many other languages working with :results value in org-babel (please see attached .spacemacs file, search for "julia" and for a bunch of other languages like Clojure and plantuml. In an org-babel block with "results value", I get something like the following

```
#+BEGIN_SRC julia :results value
rand(2,3)
#+END_SRC

#+RESULTS:
```
and `Code block returned no value` in the emacs minibuffer. With :results output, i get the following

```
#+BEGIN_SRC julia :results output
rand(2,3)
#+END_SRC

#+RESULTS:
: 2×3 Array{Float64,2}:
:  0.88332   0.805873     0.560745
:  0.571481  0.000716739  0.158407
```

so stuff is kinda working.  I would dearly love to have :results value working, and I'm sure I've done something stupid in my configuration, but I can't see what. Would you be so kind as to help?

Here is my .spacemacs and configuration [https://gist.github.com/rebcabin/d325740d2d68840cfdaac7710eebbbd5](https://gist.github.com/rebcabin/d325740d2d68840cfdaac7710eebbbd5)
```

**Comment by frederic-santos** [comment]
I think that `ob-julia` is not maintained anymore. A fork has been created [here](https://git.nixo.xyz/nixo/ob-julia), you might want to post your issue on this repo.

**Comment by rebcabin** [comment]
Oh, wow! I just checked the dates ... 6 and 7 years ago for the last commit! Ok, it's amazing that anything works at all.  Thanks for the lead!

**Comment by frederic-santos** [comment]
Yes, but it seems that your problem also happens with more recent forks (I could reproduce it). You have yet other forks out there on GitHub (another example [here](https://github.com/vincent-picaud/Julia_with_OrgMode_Example)), I do not know which one is the most advanced to date.

**Comment by rebcabin** [comment]
I tried several forks, including the one hosted in MELPA. Given your
repro, it seems that ob-julia is broken everywhere for now. I don't have
time to debug it at the present. I was tooling up for serious work with
Julia no earlier than the Fall; perhaps I may debug it then or perhaps
the authors will have fixed it. Thanks for checking on your side!

Frédéric Santos writes:

> Yes, but it seems that your problem also happens with more recent forks (I could reproduce it). You have yet other forks out there on GitHub (another example [here](https://github.com/vincent-picaud/Julia_with_OrgMode_Example)), I do not know which one is the most advanced to date.


--
Brian C. Beckman
http://www.example.com
